docs/usage/configuration-options.md

```diff
@@ -1076,7 +1076,8 @@ When this field is enabled, Renovate will abort its run if it encounters either
 
 ### authType
 
-This can be used with `token` to create a custom http `authorization` header.
+This can be only used with `token` to create a custom http `authorization` header.
+Don't set `authType=Bearer`, it's the default.
```

Comment:
```suggestion
You may use the `authType` option to create a custom HTTP `authorization` header.
For `authType` to work, you must also set your own `token`.

Do not set `authType=Bearer`: it's the default setting for Renovate anyway.
```

How about this?

Reply:
it must be clear to not use it with username and password. it won't be used there.

Reply:
```suggestion
You may use the `authType` option to create a custom HTTP `authorization` header.
For `authType` to work, you must also set your own `token`.

Do not set `authType=Bearer`: it's the default setting for Renovate anyway.
Do not set a username or password when you're using `authType`, as `authType` doesn't use usernames or passwords.
```

> it must be clear to not use it with username and password. it won't be used there.

How about something like this?